docs/sources/administration/view-server-settings.md

```diff
@@ -0,0 +1,26 @@
++++
+title = "View server settings"
+description = "How to view server settings in the Grafana UI"
+keywords = ["grafana", "configuration", "server", "settings"]
+type = "docs"
+[menu.docs]
+name = "Configuration"
+identifier = "config"
+parent = "admin"
+weight = 350
++++
+
+# View Grafana server settings
+
+If you are a Grafana server admin, then you can use the Settings tab to view the settings applied to your Grafana server by the [Configuration]({{< relref "configuration.md" >}}) file and any environmental variables.
+
+> **Note:** Only Grafana server admins can access the **Server Admin** menu. For more information about about admin permissions, refer to [Grafana server admin]({{< relref "../permissions/_index.md" >}}).
+
+## View server settings
+
+1. Log in to your Grafana server with an account that has the Grafana Admin flag set.
```

Comment:
I wasn't aware that there was such a flag! I think that this is only visible in `User Information` tab under `Server Admin` which is accessible only be administrators. So for non administrators this flag is practically invisible.
I would suggest changing to something like: `Log in to your Grafana server with an account with administrator permissions`.

Reply:
Nope, because we have two kinds of administrators: organization administrators and server administrators. And this is specific to Grafana server admins.

Reply:
Correction: Three kinds of administrators: organization admins, server admins, and team admins.

Reply:
In addition to those three, you can also have admin rights over a folder.

Reply:
Yes, but that is admin permissions, not an admin role.